lua/entities/hl_camera/ghost.lua

```diff
@@ -0,0 +1,50 @@
+function ENT:GhostCreate()
+	if IsValid(self.ACGhost) then
+		return nil
+	end
```

Comment:
Essentially this code path should never happen, but if you're checking for it anyway (which I think is a good idea), it's better to return `self.ACGhost` instead of `nil`.